src/Tests/Moryx.Model.Tests/SqliteTests.cs

```diff
@@ -0,0 +1,81 @@
+﻿using Microsoft.Extensions.Logging;
+using Moryx.Model.InMemory;
+using Moryx.Model.Sqlite;
+using Moryx.Products.Model;
+using Moryx.Runtime.Endpoints.Databases.Endpoint.Services;
+using Moryx.Runtime.Kernel;
+using Moryx.TestTools.Test.Model;
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Moryx.Model.Tests
+{
+    [TestFixture]
+    public  class SqliteTests
+    {
+        private SqliteDatabaseConfig dbConfig;
+        private SqliteModelConfigurator configurator;
+        private string datasource;
+
+        [SetUp]
+        public void Setup()
+        {
+            string databaseName = "TestDatabase";
+            datasource = $".\\db\\{databaseName}.db";
+            string connectionString = $@"Data Source={datasource};Pooling=false";
```

Comment:
Adapt the connection string here, if you change the general pooling setting to be disable on database deletion only